src/components/Modals/ProjectCardModal/ProjectCardModal.tsx

```diff
@@ -0,0 +1,198 @@
+import React, { useEffect, useState } from 'react';
+
+interface PropTypes {
+  open: boolean;
+  onClose: () => void;
+  onViewMoreProjects: () => void;
+}
+
+const ProjectCardModal: React.FC<PropTypes> = ({ open, onClose, onViewMoreProjects }) => {
+  const [answers, setAnswers] = useState<any[]>([
+    { id: 1, answerOne: "Sample answer one", answerTwo: "Sample answer two" }
+  ]);
+  const [answerOne, setAnswerOne] = useState<string>('');
+  const [answerTwo, setAnswerTwo] = useState<string>('');
+  const [isReviewMode, setIsReviewMode] = useState<boolean>(false);
+  const [submissionComplete, setSubmissionComplete] = useState<boolean>(false);
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (!answerOne || !answerTwo) {
+      alert("Please answer all the questions.");
+      return;
+    }
+    setIsReviewMode(true);
+  };
+
+  const handleFinalSubmit = () => {
+    const item = {
+      id: Math.floor(Math.random() * 1000),
+      answerOne,
+      answerTwo,
+    };
+    setAnswers((oldList) => {
+      const answers = [...oldList, item];
+      localStorage.setItem("answers", JSON.stringify(answers));
+      return answers;
+    });
+    setAnswerOne("");
+    setAnswerTwo("");
+    setIsReviewMode(false);
+    setSubmissionComplete(true);
+  };
+
+  useEffect(() => {
+    const itemsFromLocalStorage = localStorage.getItem("answers");
+    if (itemsFromLocalStorage) {
+      const parsedItemsFromLocalStorage = JSON.parse(itemsFromLocalStorage);
+      setAnswers(parsedItemsFromLocalStorage);
+    }
+  }, []);
+
+  return (
+    <div
+      className={`fixed inset-0 flex justify-center items-center transition-colors ${
+        open ? 'visible bg-black/20' : 'invisible'
+      }`}
+      onClick={onClose}
+    >
+      <div
+        className={`bg-white rounded-lg shadow p-6 transition-all max-w-md ${
+          open ? 'scale-100 opacity-100' : 'scale-110 opacity-0'
+        }`}
+        onClick={(e) => e.stopPropagation()}
```

Comment:
stopPropogation is good. is the div wrapping this the outside area of the modal and the onClick is to exit? That is good behaviour. However if you are ready to take on the challenge it is probably better handled with some kind of ref and onClick behaviour that detects if the .current is outside of the container.

Anyways keep this for now if it's too difficult